recipes/qscintilla2/bld.bat

```diff
@@ -0,0 +1,73 @@
+@echo off
+:: MSVC library for Python 2.7 is not found automaticaly if it is 
+:: installed in the user's AppData folder. Add it to PATH
+setlocal EnableDelayedExpansion
+
+if "%PY_VER%"=="2.7" (
+	set VC9DIR=%LOCALAPPDATA%\Programs\Common\Microsoft\Visual C++ for Python\9.0
+	IF EXIST !VC9DIR! ( 
+		set PATH=!VC9DIR!;"%PATH%"
+		if %ARCH% == 32 call vcvarsall.bat x86
+		if %ARCH% == 64 call vcvarsall.bat amd64
```

Comment:
Nope, not in my case at least. If you install [Microsoft Visual C++ Compiler for Python 2.7](https://www.microsoft.com/en-us/download/details.aspx?id=44266), it nowadays installs somewhere in the user's appdata folder, while conda build (only) tries to invoke vcvarsall.bat where it used to be located with the real Visual Studio 2008 installation (somewhere in Program Files). The build did not succeed on my machine without those lines at least, but this story might be different for the conda forge VMs which have the official full VC2008 tools installed (and not the version for Python)